compose/vol-type

```diff
@@ -0,0 +1 @@
+chalja
```

Comment:
How are the compose/* files getting added to this commit? Is it coming from merging a different branch?

Reply:
Believe this was an accidental commit. Removed the files altogether & updated .gitignore to prevent them from being re-added. Please pull before pushing any changes.